zerver/migrations/0425_migrate_to_user_level_email_address_visibility_setting.py

```diff
@@ -0,0 +1,36 @@
+# Generated by Django 3.2.8 on 2021-10-27 14:25
+
+from django.db import migrations
+from django.db.backends.postgresql.schema import BaseDatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+
+def migrate_to_user_level_email_address_visibility_setting(
+    apps: StateApps, schema_editor: BaseDatabaseSchemaEditor
+) -> None:
+    Realm = apps.get_model("zerver", "Realm")
+    RealmUserDefault = apps.get_model("zerver", "RealmUserDefault")
+    UserProfile = apps.get_model("zerver", "UserProfile")
+    realms = Realm.objects.all()
+    for realm in realms:
+        UserProfile.objects.filter(realm=realm, is_bot=False).update(
+            email_address_visibility=realm.email_address_visibility
+        )
```

Comment:
Maybe add a comment noting that database-leverl default value, for bots, of `EMAIL_ADDRESS_VISIBILITY_EVERYONE` is correct.

BTW, can you double-check that bots get `EMAIL_ADDRESS_VISIBILITY_EVERYONE` during account creation even if the realm's new user defaults setting is something different?

Reply:
> Maybe add a comment noting that database-leverl default value, for bots, of EMAIL_ADDRESS_VISIBILITY_EVERYONE is correct.

Aded comment.

Also, the `email_address_visibility` was not set to `EVERYONE` for bots, updated it now, so thanks for catching this.

Reply:
Great, thanks!